static/app/views/dashboardsV2/widgetLibrary/data.tsx

```diff
@@ -166,3 +166,15 @@ export const DEFAULT_WIDGETS: Readonly<Array<WidgetTemplate>> = [
     ],
   },
 ];
+
+export const getTopNConvertedDefaultWidgets = () =>
```

Comment:
should we move this function to a new `utils.tsx `file or just add to the `widgetLibrary` file?

Reply:
I was thinking about that, but I think this function is small and it's nice to have it close to the data. It should be short-lived though once we fully GA we can just convert the relevant values in `DEFAULT_WIDGETS`